demo/scenario/script

```diff
@@ -17,10 +17,37 @@ submit Auctioneer1 createAuction
 
 # clock-rounds ...
 submit Auctioneer1 startNextRound
-wait "enter A-Telecom bid manually ...  (Hit any key to continue)"
+submit A-Telecom submitClockBid
 submit B-Net submitClockBid
+submit C-Mobile submitClockBid
 submit Auctioneer1 endRound
 
+wait "End Round 1  (Hit any key to continue)"
+
+submit Auctioneer1 startNextRound
+submit A-Telecom submitClockBid
+submit B-Net submitClockBid
+submit C-Mobile submitClockBid
+submit Auctioneer1 endRound
+
+wait "End Round 2  (Hit any key to continue)"
+
+submit Auctioneer1 startNextRound
+wait "enter C-Mobile bid manually ...  (Hit any key to continue)"
```

Comment:
is this "enter manually" intended?
If I don't enter anything, or I enter the actual C-mobile bid, there are different outcomes.

Reply:
I think we need a comment here what C-mobile has to enter now in order to continue the scenario as intended.